subsys/bluetooth/controller/crypto.c

```diff
@@ -10,19 +10,17 @@
 #include <stdint.h>
 #include <soc.h>
 #include <logging/log.h>
+#include <ble_controller_soc.h>
 
-#include "nrf_error.h"
-
-#include "ble_controller_soc.h"
-#include "blectlr_util.h"
+#include "nrf_errno.h"
+#include "multithreading_lock.h"
 
 #define LOG_MODULE_NAME ble_controller_crypto
 LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_DBG);
 
-
 int bt_rand(void *buf, size_t len)
 {
-	if (buf == NULL) {
+	if (len > UINT8_MAX || buf == NULL) {
```

Comment:
Why did you change bt_rand?

Reply:
This is probably leftover after @PavelVPV changes in his PR. This condition should probably be removed now this condition was valid when ble_controller API was used directly.